app/models/iam_user_identity.rb

```diff
@@ -38,6 +38,10 @@ def self.build_from_iam_profile(iam_profile)
     identity
   end
 
+  def multifactor
+    loa[:current].try(:to_i) == LOA::THREE
```

Comment:
think this can use ruby's [safe navigator](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)

Reply:
on it